CMakeLists.txt

```diff
@@ -262,6 +262,9 @@ else()
     message(STATUS "Development: Unit-tests are disabled")
 endif()
 
+# Copy openomf.bk to destination resources directory to ease development setup.
+file(COPY resources/openomf.bk DESTINATION resources)
```

Comment:
Will this work if someone builds to the project root ? It will try to copy in-place ?

Reply:
I tested this, and it did not break, so cmake . worked.